src/method_ilan.jl

```diff
@@ -166,32 +172,71 @@ function ilan(
 
         orthogonalize_and_normalize!(view(V,:,1:k),view(V,:,k+1), view(HH,1:k,k), orthmethod)
 
+
         # extract eigenpair approximation
         if ((rem(k,check_error_every)==0)||(k==m))&&(k>2)
```

Comment:
Can the `k>2`-condition be removed? It has been removed from iar etc. For a user who wants to plot the error, this basically prevents you from accessing the error in iteration 1 and 2.